include/deal.II/matrix_free/operators.h

```diff
@@ -815,7 +816,7 @@ namespace MatrixFreeOperators
   template <int dim, int fe_degree, int n_components, typename Number>
   void
   MassOperator<dim, fe_degree, n_components, Number>::
-  local_apply_cell (const MatrixFree<dim,Number>                     &data,
```

Comment:
alternatively one could use it below in `FEEvaluation`. But it would amount to the same thing here